src/environment.F

```diff
@@ -1062,6 +1072,7 @@ SUBROUTINE diag_setup_library(globenv)
                      elpa_qr=globenv%elpa_qr, &
                      elpa_print=globenv%elpa_print, &
                      elpa_qr_unsafe=globenv%elpa_qr_unsafe, &
+                     dlaf_neigvec_min_input=globenv%dlaf_neigvec_min, &
```

Comment:
Does the `_input` suffix belong here?

Reply:
Yes, `dlaf_neigvec_min_input` is the name of the input variable, which sets the `dlaf_neigvec_min` variable in the `cp_fm_diag` module.

https://github.com/RMeli/cp2k/blob/a76d27d83272c686d563b8d44cfe03a86a81645a/src/fm/cp_fm_diag.F#L128-L129